PKGBUILD

```diff
@@ -113,7 +128,7 @@ build() {
   export TF_NEED_ROCM=1
   ./configure
   bazel \
-    build --config=mkl --config=avx2_linux -c opt \
+    build -s --config=mkl --config=avx2_linux -c opt \
```

Comment:
Im not that familiar with bazel, what does `-s` do?

Reply:
It's a verbose flag, forgot to remove it